rules/klayout/drc/testing/Makefile

```diff
@@ -23,16 +23,16 @@ Testing_DIR ?= $(shell pwd)
 
 all : clean test-DRC
 
-test-DRC: test-DRC_switch  test-DRC_SC  test-DRC_main
+test-DRC: test-DRC-switch  test-DRC-SC  test-DRC-main
```

Comment:
The changes to these `Makefile` should probably be a separate commit?

Reply:
@mithro these changes are done by @umarcor to make it cleaner for github actions. Doesn't affect the main functionality from my end.

Reply:
See https://github.com/google/globalfoundries-pdk-libs-gf180mcu_fd_pr/pull/28.

Reply:
#28 looks good. Will merge if CI is green.